Description: specify stack direction for aarch64
 The generic test does not work with cross compilation.
Author: Helmut Grohne <helmut@subdivi.de>
Last-Modified: 2015-01-10
Bug-Debian: https://bugs.debian.org/775050
Forwarded: no
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- libsigsegv.orig/m4/stack-direction.m4
+++ libsigsegv/m4/stack-direction.m4
@@ -16,7 +16,7 @@
       aarch64* | \
       alpha* | \
       arc | \
-      arm* | strongarm* | xscale* | \
+      aarch64 | arm* | strongarm* | xscale* | \
       avr | avr32 | \
       bfin | \
       c1 | c2 | c32 | c34 | c38 | \
